devtools/dump_devinfo.py

```diff
@@ -452,9 +481,196 @@ async def get_smart_fixture(device: SmartDevice, batch_size: int):
     sw_version = sw_version.split(" ", maxsplit=1)[0]
 
     save_filename = f"{model}_{hw_version}_{sw_version}.json"
-    copy_folder = "kasa/tests/fixtures/smart/"
+    copy_folder = SMART_FOLDER
     return save_filename, copy_folder, final
 
 
+@cli.command(name="generate-supported")
+async def generate_supported():
+    """Generate the SUPPORTED.md from the fixtures."""
+    supported = {
+        PLUGS: {"kasa": {}, "tapo": {}},
+        POWER_STRIPS: {"kasa": {}, "tapo": {}},
+        WALL_SWITCHES: {"kasa": {}, "tapo": {}},
+        BULBS: {"kasa": {}, "tapo": {}},
+        LIGHT_STRIPS: {"kasa": {}, "tapo": {}},
+    }
+    _get_iot_supported(supported)
+    _get_smart_supported(supported)
+
+    _update_supported_file(SUPPORTED_FILENAME, supported, False)
```

Comment:
```suggestion
    _update_supported_file(SUPPORTED_FILENAME, supported, False)
```
The last flag should probably be a kwarg-only, now it's a mystery what it does w/o finding the relevant function :-) Not a big deal in devtools though.

Reply:
Done